.github/pull_request_template.md

```diff
@@ -1,23 +1,34 @@
 # Description
-<!-- Briefly describe the feature or bug that your pull request addresses -->
+<!-- Briefly describe the feature or bug that your pull request addresses, 1-2 sentences. -->
 
-# Check List
 
-- [ ] Are you changing anything with the public API?
-- [ ] Are your changes backwards compatible with previous SDK Versions?
-- [ ] Have you tested this change on real device?
-- [ ] Have you added unit test coverage for your changes?
-- [ ] Have you verified that your changes are compatible with all Android versions the SDK currently supports?
+## Due Diligence
+<!-- Best practices before submitting, add additional notes below -->
+- [ ] I have tested this on an emulator and/or a physical device.
+- [ ] I have added sufficient unit/integration tests of my changes.
+- [ ] I have adjusted or added new test cases to team test docs, if applicable.
+- [ ] I am confident these changes are compatible with all Android versions the SDK currently supports.
 
 
+## Release/Versioning Considerations
+<!-- Help determine how this should be categorized for release, add additional notes below. -->
+- [ ] `Patch` Contains internal changes or backwards-compatible bug fixes.
+- [ ] `Minor` Contains changes to the public API. 
+- [ ] `Major` Contains **breaking** changes.
+- [ ] Contains readme or migration guide changes.
+  - If so, please merge to a feature branch so documentation updates only go live upon official release.
+- [ ] This is planned work for an upcoming release.
+  - If no, author or reviewer should account for this in a release plan, or describe why not below.
+
 ## Changelog / Code Overview
 <!-- What was changed / added / removed and why. Attach screenshots or other supporting materials -->
 
 
 ## Test Plan
-<!-- How was this code tested / How should reviewers test it? -->
+<!-- Provide reproducible testing steps. Link any artifacts, recordings, spreadsheets, etc. -->
```

Comment:
In light of the app/fender discussions, I actually wanted to clarify that (in my opinion...) the onus is on the author to provide test steps and outcomes, and the reviewer to _verify_ but not necessarily _run the tests_. I think repeating the tests should be up to the reviewer's discretion.